mapflow/mapflow.py

```diff
@@ -2448,6 +2449,19 @@ def download_results_file(self) -> None:
             return
         self.result_loader.download_results_file(pid=pid)
 
+    def download_aoi_file(self) -> None:
+        """
+        Download area of interest and save to a geojson file
+        """
+        processing = self.selected_processing()
+        if not processing:
+            return
+        pid = processing.id_
+        if pid not in self.processing_history.finished:
+            self.alert(self.tr("Only the areas of correctly finished processing can be loaded"))
```

Comment:
No, actually this is user uploaded data and he can access it even before completion
So, no need to restrict it